### DISCUSSION TITLE
User & User Account Definition Review

### DISCUSSION THREAD

**Discussion by dsdr0** [d3fend/d3fend-ontology#136] [General]
**User - Object Properties** & **User Account - Object Properties** 
States user is a person [or agent] but believe it should explicitly include Non-Person Entities (NPEs), also known as non-users in some circles. There is an important distinction between the two sets of users within many policies/requirements. 

** User Account - Object Properties ** 
To include NPEs and lack of "login", suggest changing "login" language to authenticate. I would either remove login example or expand examples to include a web application example (or something else).

**Comment by hack-sentinel** [comment]
Good/needed add.  Note for zero-trust additions coming: This ticket is aligned with expanded definition and specification for zero-trust ZT (need to handle NPEs) and general authentication; team should 
make sure to relate this issue to upcoming ZT tickets and align any model refactoring for zero trust architecture (ZTA) issues (and vice versa, for single model.)

**Comment by netfl0** [comment]
opened issue to address
